exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/PlannerSettings.java

```diff
@@ -241,6 +244,10 @@ public long getInSubqueryThreshold() {
     return options.getOption(IN_SUBQUERY_THRESHOLD);
   }
 
+  public boolean isAnsiQuotesEnabled() {
```

Comment:
For this reason maybe it makes sense to implement not boolean option ANSI_QUOTES, but some string option, to turn on necessary lexical policy.  
I raised this question in jira [DRILL-3510](https://issues.apache.org/jira/browse/DRILL-3510?focusedCommentId=15780561&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15780561)